metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -169,6 +170,7 @@ class MetalsLanguageServer(
       config.icons,
       statusBar
     )
+    documentSymbolProvider = new DocumentSymbolProvider(buffers)
```

Comment:
Looks like this can be a `val` at the top of the class, `buffers` does not depend on `workspace`

Reply:
Yes, I've kept it like this for consistency, but I can use a `val`